commons/constants.py

```diff
@@ -481,3 +481,8 @@ class Sizes:
 CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
 "the CreateBucket operation: Your previous request to create the named bucket succeeded and "
 "you already own it.")
+RGW_ERR_NO_SUCH_UPLOAD = ("An error occurred (NoSuchUpload) when calling the UploadPart operation: Unknown")
```

Comment:
Remove all redundant parentheses.

Reply:
from PEP8: 
https://peps.python.org/pep-0008/#multiline-if-statements

The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
That is for condtional statements.

Reply:
pls share link for PEP8 standard for multiple line strings where it mentions no use of () around string which go beyond single line to avoid E501

Reply:
Redundant parentheses allowed as per https://google.github.io/styleguide/pyguide.html